schemas/stsci.edu/asdf/0.1.0/asdf-schema.yaml

```diff
@@ -0,0 +1,46 @@
+%YAML 1.1
+---
+$schema: "http://json-schema.org/draft-04/schema"
+id: "http://stsci.edu/schemas/asdf/0.1.0/asdf-schema"
+title:
+  ASDF schema
+description: |
+  A metaschema extending YAML Schema and JSON Schema to add support
+  for some ASDF-specific checks, related to nd-arrays.
```

Comment:
So, I definitely get your point (why include a tool for something we can do already), but it sure makes things verbose as I try to convert things over on the jwst_lib.models end.  The exact incantation (equivalent to ndim) is:

```
allOf:
   - $ref: <...>
   - properties:
      shape:
         minItems: 2
         maxItems: 2
```

vs.

```
ndim: 2
```

(since in the latter case, the fact that it's an array is also conveniently implied).